### ISSUE TITLE
Accessibility-> Missing an option to Scroll to top of the page

### ISSUE THREAD

**Issue by manuji** [bcgov/nr-rec-resources#594]
**Describe the bug**
There should be a way for users to easily scroll to the top of the page. This issue fails the [2.4.1 Bypass Blocks (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/bypass-blocks) criterion.

**To Reproduce**
1. Go to 'https://d1vrz2gt4nmfe5.cloudfront.net/'
2. Check if the two pages of the new beta site has options to scroll to the top of the page

**Expected behavior**
Provide an option to "scroll to top" to quickly navigate to the beginning of a page, bypassing potentially long sections of text or content. 

**Screenshots**

<img width="1467" alt="Image" src="https://github.com/user-attachments/assets/b60e9f13-3104-4c2a-bb90-35fb45b26c8c" />

**Comment by airliah** [comment]
looks good to me!
